src/viam/sdk/common/utils.cpp

```diff
@@ -58,16 +59,14 @@ std::string bytes_to_string(const std::vector<unsigned char>& b) {
     return img_string;
 };
 
-std::chrono::time_point<long long, std::chrono::nanoseconds> timestamp_to_time_pt(
-    const google::protobuf::Timestamp& timestamp) {
+time_point timestamp_to_time_pt(const google::protobuf::Timestamp& timestamp) {
     const std::chrono::seconds seconds(timestamp.seconds());
     const std::chrono::nanoseconds nanos(timestamp.nanos());
-    return std::chrono::time_point<long long, std::chrono::nanoseconds>(
-        std::chrono::duration_cast<std::chrono::system_clock::duration>(seconds) + nanos);
+    return time_point(std::chrono::duration_cast<std::chrono::system_clock::duration>(seconds) +
```

Comment:
Elsewhere `steady_clock`, is `system_clock` here intentional?

Reply:
Sounds good! Will revert to `long long`, open a new ticket to revisit, and then merge.